libs/design/guides/foundations/theming.md

```diff
@@ -1,17 +1,45 @@
 # Theming
-Daffodil's theming capabilities enables you to customize `@daffodil/design` components to reflect your brand. A theme consists of custom color configurations that will work in dark and light themes.
+Daffodil's theming capabilities enables you to customize `@daffodil/design`'s component color styles to reflect your brand or product.
```

Comment:
Small change but I think this should be "enable" instead of "enables"

Reply:
updated